kmapper/kmapper.py

```diff
@@ -637,11 +638,14 @@ def visualize(
         color_function : list or 1d array
             A 1d vector with length equal to number of data points used to build Mapper. Each value should correspond to a value for each data point and color of node is computed as the average value for members in a node.
 
+        colorscale : list
+            Specify the colorscale to use. See visuals.colorscale_default.
+
```

Comment:
point the reader to the default colorscale as an example

Reply:
It'd be nice to specify the full path to get to that function so it's a bit clearer. Maybe add a markdown style link like
`[visuals.colorscale_default]( https://github.com/scikit-tda/kepler-mapper/blob/51dde7d0469feeb7d76ba1932bae4ba7cc71e970/kmapper/visuals.py#L10)`?  I'm not sure if that's the right method, but I could imagine newcomers getting hung up on how to find the `colorscale_default`.